nevergrad/optimization/optimizerlib.py

```diff
@@ -2561,3 +2561,44 @@ def recommend(self) -> p.Parameter:
 @registry.register
 class NGOpt(NGOpt10):
     pass
+
+
+# Variants dedicated to multiobjective optimization by multiple monoobjective optimization.
+@registry.register
+class NGOpt10_16(CM):
```

Comment:
Please create a Configured optimizer instead, with the number of optimizer and the class of optimizer (or just a list of OptCls if you expect you would use different algorithms at some point).

This way it will be way more generic

Reply:
Done.